tests/tts_tests/test_vits_speaker_emb_train_upsampling_interpolation_approach.py

```diff
@@ -0,0 +1,90 @@
+import glob
```

Comment:
We don't need these costly tests. 

Just write unittests in ```test_vits.py```

Reply:
Indeed, good catch, Done :)